Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs

```diff
@@ -150,6 +150,19 @@ public FlattenModeType FlattenAxis
             }
         }
 
+        [SerializeField]
+        [Tooltip("Whether scale the flattened axis when uniform scale is used.")]
+        private bool uniformScaleOnFlattenedAxis = true;
```

Comment:
I am not sure if this value should be set to true, it might be better to set it to false by default.  If a user selects a specific axis to flatten, I think they would not expect the scaling to be uniform by default after indicating their intention to flatten a specific axis.

Reply:
I have mixed feelings towards this option... On the one hand I think it should default to false, but defaulting to true will keep the current (though not necessarily preferable but at least not breaking) behavior.

Reply:
Wherever possible, we should strive to maintain the current behavior as the default so we do not break customer applications when they update.

If a non-default setting is optimal, we should document the setting, what makes it preferable and why. When we update our major version, we can change the behavior without fear.